mangaki/mangaki/models.py

```diff
@@ -572,7 +573,7 @@ class FAQEntry(models.Model):
     def __str__(self):
         return self.question
 
-class Trope(models.Model): 
+class Trope(models.Model):
```

Comment:
When it's possible, try to avoid large whitespace/code style changes because they're not logically related to the contents of the PR and may hinder the review (I say this, but I usually do it, not on purpose).

Though, it's really okay on this PR.